### PR TITLE
Updated to support Laravel 9.

### PR DESCRIPTION
**Pull request by DmitrySidorenkoShim**

### CODE REVIEW

File: src/Commands/SitemapCommand.php [hunk 1]
```diff
@@ -64,7 +71,7 @@ protected function crawlWebsite($url)
 
         // Add a URI discoverer. Without it, the spider does nothing.
         // In this case, we want <a> tags and the canonical link
-        $spider->getDiscovererSet()->set(new XPathExpressionDiscoverer("//a|//link[@rel=\"canonical\"]"));
+        $spider->getDiscovererSet()->set(new XPathExpressionDiscoverer("//div[@rel=\"canonical\"]//a"));
```
Comment (spekulatius) [inline src/Commands/SitemapCommand.php]: Hey @DmitrySidorenkoShim,

what is the plan here? The selector wasn't working for you?

**Comment by DmitrySidorenkoShim** [comment]
@spekulatius 
I have no good example of `robots.txt` for testing
mine looks like this:
```
User-agent: *
Disallow:
```

do you have something to test?

I changed it because I had an error for `"//a|//link[@rel=\"canonical\"]"`
and I found this description in [mvdbos](https://github.com/mvdbos)/[php-spider](https://github.com/mvdbos/php-spider) package readme

the command is working
but output looks like this right now:
```
$ php artisan sitemap:generate
Starting site crawl...
Loading robots.txt from http://laravel-nine.local/robots.txt
Enqueued:  1
Skipped:   0
Failed:    0
Persisted: 0

Resources:
Writing sitemap.xml into public directory...
Sitemap generation completed.
```

File: src/Commands/SitemapCommand.php (hunk 1 above)
Comment (spekulatius) [reply src/Commands/SitemapCommand.php]: @DmitrySidorenkoShim

**Comment by spekulatius** [comment]
Hey @DmitrySidorenkoShim 

I still think the change on the selector shouldn't be included. Can you undo it? Then we can merge it and get this released.

Cheers,
Peter

**Comment by DmitrySidorenkoShim** [comment]
@spekulatius 

I changed back this
thanks for the feedback
and I hope you can merge this PR

Best regards 
Dimon

**Comment by spekulatius** [comment]
Thanks for your contribution and patience @DmitrySidorenkoShim

**Pull request merged**